src/features/collection/components/cards/ExperienceModalCard.tsx

```diff
@@ -0,0 +1,211 @@
+'use client';
+
+import Button from '@/components/Button/Button';
+import Tag from '@/components/Tag/Tag';
+import { Flex, Modal, ModalBody, ModalContent, ModalHeader, ModalOverlay } from '@chakra-ui/react';
+import Image from 'next/image';
+import { Capacity } from '../../types';
+import cardImage from '../../../../../public/images/card1.png';
+import { useState } from 'react';
+import TextAreaField from '@/components/Input/TextAreaField/TextAreaField';
+import MotionBox from './MotionBox';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  // Todo: Data
+  period: string;
+  title: string;
+  summaries: string[];
+  capabilities: Omit<Capacity, 'count'>[];
+};
+
+const ExperienceModalCard = (props: Props) => {
+  const { isOpen, onClose, period, title, summaries, capabilities } = props;
```

Comment:
```suggestion
const ExperienceModalCard = ({ isOpen, onClose, period, title, summaries, capabilities }: Props) => {
```

구조 분해 할당해서 코드 한 줄 줄여봅시다..!